styles.css

```diff
@@ -0,0 +1,52 @@
+body {
+  background-color: #ebe4e4;
+  font-family: Arial, Helvetica Neue, Helvetica, sans-serif; 
+  margin: 0;
+}
+
+header {
+  background-color: #e62d27;
+  text-align: center;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  color: #FFF;
+  margin-bottom: 30px;
+  grid-column-start: 1;
```

Comment:
Good job trying to use css grid. But please add a display: grid properties for your css style to work.

Reply:
Kudos for this one! You're explaining in a polite way that in order for the grid to work, you must have a `display: grid` property in a parent element! 💯